cpp/src/parquet/encoding.cc

```diff
@@ -2883,7 +2989,26 @@ class RleBooleanDecoder : public DecoderImpl, virtual public BooleanDecoder {
   int DecodeArrow(int num_values, int null_count, const uint8_t* valid_bits,
                   int64_t valid_bits_offset,
                   typename EncodingTraits<BooleanType>::Accumulator* out) override {
-    ParquetException::NYI("DecodeArrow for RleBooleanDecoder");
+    if (null_count != 0) {
+      ParquetException::NYI("RleBoolean DecodeArrow with null slots");
```

Comment:
Is this a TODO or misuse for `null_count != 0`?

Reply:
Yes...Here implement it just for testing. And later `DecodeArrow with null slots` not have todo. I may do it later